sitemap.html

```diff
@@ -7,7 +7,7 @@
 			<meta name="generator" content="Mystemapgenerator.com HTML Sitemap" />
 			<meta charset="utf-8" />
 			<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=yes" />
-			<link rel="stylesheet" href="sitemapStyles.css">
+			<link rel="stylesheet" href="./css/sitemapStyles.css">

```

Comment:
In the project we are following this `css/sitemapStyles.css`, so can you remove the initial `./` so that it will be same across all files.

Reply:
I have simplified the path with respect to your need.

Reply:
@harikesh409 I hope that's ok now.